src/Categories/Functor/Bialgebra.agda

```diff
@@ -0,0 +1,66 @@
+{-# OPTIONS --without-K --safe #-}
+
+module Categories.Functor.Bialgebra where
+
+open import Level
+open import Categories.Category using (Category)
+open import Categories.Functor
+open import Categories.Functor.Algebra
+open import Categories.Functor.Coalgebra
+open import Categories.NaturalTransformation
+
+{-
+given two endofunctors T and F on a category C, we can define a (T,F)-bialgebra to be an object of C equipped with the structure of a T-algebra and an F-coalgebra, i.e. just a triple (A,a,c), where a:TA→A and c:A→FA.
+μ is a distributive law (of T over F): a natural transformation μ:TF→FT.
+A μ-bialgebra is a (T,F)-bialgebra (A,a,c) such that c∘a=Fa∘μ_A∘Tc
+[category theory - How are F-bialgebras defined? - Mathematics Stack Exchange](https://math.stackexchange.com/questions/3057781/how-are-f-bialgebras-defined#answer-3057859)
+-}
+
+private
+  variable
+    o ℓ e : Level
+
+module _ {C : Category o ℓ e} where
+
+  module _ (T : Endofunctor C) (F : Endofunctor C) where
```

Comment:
`Categories.NaturalTransformation` imports `Categories.Functor.Properties`. Perhaps it's better off in `Categories.NaturalTransformation.Properties`? The reason why I didn't want to place it somewhere very globally is because Distributive Laws are generally defined for monads and since I'm only using them for functors here I don't require the usual extra properties: https://ncatlab.org/nlab/show/distributive+law

Reply:
Ah I forgot to write it out: The problem is that putting it in `Categories.Functor.Properties` creates an import cycle

Reply:
Ah, now *that* is a real problem! `Categories.Functor.DistributiveLaw` it is then?